boards/risc-v/esp32c3/esp32c3-devkit/src/esp32c3_bringup.c

```diff
@@ -161,6 +165,17 @@ int esp32c3_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_SPI_DRIVER
```

Comment:
maybe in this case using: #if defined(CONFIG_SPI_DRIVER) && defined(CONFIG_ESP32C3_SPI2) will make the code cleaner

Reply:
Thanks for the suggestion! Fixed.

Reply:
Thank you!